### PR TITLE
testing/rubberband: new aport

### PR DESCRIPTION
**Pull request by 9001**
audio stretching and key-shifting library

### CODE REVIEW

File: testing/rubberband/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+pkgname=rubberband
+pkgver=1.8.2
+pkgrel=0
+pkgdesc="Audio time-stretching and pitch-shifting"
+url="https://breakfastquay.com/rubberband/"
+arch="all"
+license="GPL-2.0-or-later"
+makedepends="vamp-sdk-dev openjdk8 libsamplerate-dev libsndfile-dev fftw-dev ladspa-dev"
+subpackages="$pkgname-dev $pkgname-java:_java $pkgname-vamp:_vamp $pkgname-ladspa:_ladspa"
+source="https://breakfastquay.com/files/releases/rubberband-1.8.2.tar.bz2"
```
Comment (ncopa) [inline testing/rubberband/APKBUILD]: you should use `$pkgver` in source instead of literal `1.8.2`.

**Comment by maxice8** [comment]
Sadly this depends on a package in unmaintained/

**Comment by tcely** [comment]
> Sadly this depends on a package in unmaintained/

Which packages?

**Comment by maxice8** [comment]
ladspa

**Comment by tcely** [comment]
@9001 please move `ladspa` from `unmaintained` to `testing` as part of this pull request if you would like it merged.

**Comment by maxice8** [comment]
256 days since last interaction, can be closed.

**Comment by maxice8** [comment]
@Ikke

**Comment by Ikke** [comment]
Closing due to inactivity. Feel free to resubmit if you are interested getting this in again.

**Pull request closed without merging**